apprise/plugins/NotifyMailgun.py

```diff
@@ -61,7 +61,7 @@
 from ..AppriseLocale import gettext_lazy as _
 
 # Used to validate your personal access apikey
-VALIDATE_API_KEY = re.compile(r'^[a-z0-9]{32}-[a-z0-9]{8}-[a-z0-9]{8}$', re.I)
+# VALIDATE_API_KEY = re.compile(r'^[a-z0-9]{32}-[a-z0-9]{8}-[a-z0-9]{8}$', re.I)
```

Comment:
Don't bother commenting it out... just remove it entirely.

To avoid triggering on a pep8 error, remove the `import re` reference too.